src/lib.rs

```diff
@@ -88,6 +88,12 @@ impl Clay {
         }
     }
 
+    pub fn enable_debug_mode(&self) {
```

Comment:
Maybe it's better to send in a `bool` here? In case the user wants to enable debug mode and then turn it off again later.

Reply:
Yeah forgot that was a possibility